app/validators/slack-integration.js

```diff
@@ -5,7 +5,7 @@ export default BaseValidator.create({
     properties: ['url'],
 
     url(model) {
-        let url = model.get('url');
+        let url = model.get('url').trim();
```

Comment:
It's possible that the url value is `null` or `undefined`, there should be a guard to protect against that when assuming string methods are available, eg:

```javascript
let url = (model.get('url') || '').trim();
```

Reply:
or alternatively:

```javascript
let url = model.getWithDefault('url', '').trim();
```

Reply:
```
let url = model.getWithDefault('url', '').trim();
```
Good point, thanks, I'll add that.